<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VowelCat</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>9</number>
    </property>
    <property name="leftMargin">
     <number>12</number>
    </property>
    <property name="topMargin">
     <number>12</number>
    </property>
    <property name="rightMargin">
     <number>12</number>
    </property>
    <property name="bottomMargin">
     <number>12</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="hello">
      <property name="spacing">
       <number>9</number>
      </property>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QCustomPlot" name="customPlot" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>640</width>
             <height>480</height>
            </size>
           </property>
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>266</x>
              <y>10</y>
              <width>341</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>English</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="beginButton">
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="recordButton">
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="stopButton">
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="playButton">
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="pauseButton">
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="endButton">
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QScrollArea" name="specScroll">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>96</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>96</height>
         </size>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="widget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>801</width>
           <height>94</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="specContainer">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>9</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="QGroupBox" name="englishGroupBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>260</width>
          <height>260</height>
         </size>
        </property>
        <property name="title">
         <string>                           English</string>
        </property>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>i</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_2">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>20</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>y</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_3">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>20</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>ɨ</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_4">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>20</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>ʉ</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_5">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>20</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>ɯ</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_6">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>20</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>u</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_7">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>ɪ</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_8">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>60</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>ʏ</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_9">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>60</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_10">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>60</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_11">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>60</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_12">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>60</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>ʊ</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_13">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>e</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_14">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>100</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>ø</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_15">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>100</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>ɘ</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_16">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>100</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>ɵ</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_17">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>100</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>ɤ</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_18">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>100</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>o</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_19">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>140</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>ɛ</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_20">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>140</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>œ</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_21">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>140</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>ɜ</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_22">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>140</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>ɞ</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_23">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>140</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>ʌ</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_24">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>140</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>ɔ</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_25">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>180</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>æ</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_26">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>180</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_27">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>180</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>ɐ</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_28">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>180</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_29">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>180</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_30">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>180</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_31">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>220</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>a</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_32">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>220</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>ɶ</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_33">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>220</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>ä</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_34">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>220</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_35">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>220</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>ɑ</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_36">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>220</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>ɒ</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>9</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="QPushButton" name="addSymbolButton">
          <property name="text">
           <string>Add Symbol:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>24</height>
           </size>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="toggleChart">
        <property name="text">
         <string>Toggle Accent Profile</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionInvertAxis"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuVowelSymbols">
    <property name="title">
     <string>Symbols</string>
    </property>
    <addaction name="actionOpenSymbols"/>
    <addaction name="actionSaveSymbols"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuVowelSymbols"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionInvertAxis">
   <property name="text">
    <string>Invert Vertical Axis</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionResetPlot">
   <property name="text">
    <string>Reset Plot</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionRecordStop">
   <property name="text">
    <string>Record/Stop</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionPlayPause">
   <property name="text">
    <string>Play/Pause</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionSkipBeginning">
   <property name="text">
    <string>Skip to Beginning</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionSkipEnd">
   <property name="text">
    <string>Skip to End</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionOpenSymbols">
   <property name="text">
    <string>Load...</string>
   </property>
  </action>
  <action name="actionSaveSymbols">
   <property name="text">
    <string>Save...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
